src/node/historical_queries.h

```diff
@@ -17,6 +17,12 @@
 #include <memory>
 #include <set>
 
+#ifdef ENABLE_HISTORICAL_VERBOSE_LOGGING
+#  define HISTORICAL_LOG(...) LOG_INFO_FMT(__VA_ARGS__)
```

Comment:
I kept adding and removing some verbose logging in this file. I don't think it's useful to keep at `TRACE`, but don't want to lose it entirely, so instead its disabled by this extra switch, intended to be turned on locally for debugging the unit tests. This is a great argument for custom/scoped loggers!

Reply:
Yes, that's definitely something we should look into after 2.0.